_search-plugins/cross-cluster-search.md

```diff
@@ -297,3 +297,56 @@ curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-passwo
   }
 }'
 ```
+
+## OpenSearch behind a proxy
+You can configure cross-cluster search on a cluster running behind a proxy. There are many ways to configure a reverse proxy and various proxies to chose from. The following example demonstrates the basic NGINX reverse proxy configuration without TLS termination. OpenSearch is expected to be running with both transport and HTTP TLS encryption enabled. For more information about configuring TLS encryption, see [Configuring TLS certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls/).
+
+### Proxy configuration
+The basic NGINX configuration for HTTP and transport communication follows:
+
+```
+stream {
+    upstream opensearch-transport {
+        server <opensearch>:9300;
+    }
+    upstream opensearch-http {
+        server <opensearch>:9200;
+    }
+    server {
+        listen 8300;
+        ssl_certificate /.../opensearch-2.12.0/config/esnode.pem;
```

Comment:
Nit: Can we replace `2.12.0` with a generic key? something like 'VERSION'?

Reply:
@AntonEliatra : This will replace the variable with whatever version of the docs they are in: {{site.opensearch_version}}. 

So, you could have `ssl_certificate /.../{{site.opensearch_version}}/config/esnode.pem;`

Reply:
Thanks @hdhalter, thats updated now